src/config/mod.rs

```diff
@@ -1556,6 +1573,14 @@ impl DbConfig {
         self.writecf.validate()?;
         self.raftcf.validate()?;
         self.titan.validate()?;
+        if self.writecf.write_buffer_limit.is_some()
+            || self.defaultcf.write_buffer_limit.is_some()
+            || self.raftcf.write_buffer_limit.is_some()
+        {
+            return Err(
```

Comment:
We can support them all, as long as `write_buffer_stall_ratio` is 0.

Reply:
It's better to not support now? Expose this when we have use cases and relevant test.

Reply:
The code is very simple (one modification in `build_cf_resources`) compared to the benefit. When you forbid a configuration it would be hard to work around it when needed.

Reply:
Okay，done